mne/epochs.py

```diff
@@ -1039,6 +1039,9 @@ def plot_topo_image(self, layout=None, sigma=0., vmin=None, vmax=None,
             matplotlib borders style to be used for each sensor plot.
         fig_facecolor : str | obj
             The figure face color. Defaults to black.
+        fig_background : None | numpy ndarray
+            A background image for the figure. This must work with a call to
+            plt.imshow. Defaults to None.
```

Comment:
I don't like the phrasing "This must work with ..." ... Maybe something like "must be a valid input to matplotlib.pyplot.imshow"?